build-system/tasks/visual-diff.js

```diff
@@ -60,8 +60,9 @@ gulp.task(
         'master': '  Includes a blank snapshot (baseline for skipped builds)',
         'verify': '  Verifies the status of the build ID in ./PERCY_BUILD_ID',
         'skip': '  Creates a dummy Percy build with only a blank snapshot',
+        'headless': '  Runs Chrome in headless mode',
```

Comment:
Should we default to headless? If it's not headless, the user risks interrupting the test accidentally (imaging you start running the test in your terminal, switch to another window and begin typing. Then a Chrome window opens, takes focus, and you just modified the results because you thought you're typing on the other window that just lost focus...)

What are the benefits of _not_ defaulting to `--headless`?

Reply:
I agree that headless mode is useful. For consistency though, I think we should change this default when we change the default mode of `gulp test`. Let's explore that in another PR. I think you opened an issue, right?

Reply:
In addition, the Chrome window does not take focus in non-headless mode. I find it very useful to be able to see what the page looks like while developing new tests.

Reply:
I did now: #13402